### DISCUSSION TITLE
Is there a limit for the length of docker tags?

### DISCUSSION THREAD

**Discussion by jorisjumanne** [goharbor/harbor#19908] [Q&A]
I seem to have found a limit for the length of tags of docker images to be shown in Harbor. A very long tag name of around 70 characters doesn't show up in the harbor UI, where a tag length of 15 characters does. 

Does a limit apply and, if so, what is this limit? I can then take it into account while settings tags for my images and no longer have tagless images in Harbor.

**Comment by wy65701436** [comment]
I would like to convert this as a bug, and we will have a debug.

**Comment by jorisjumanne** [comment]
Dear @wy65701436 I think this issue can be closed. I found out that images were being untagged because new versions were uploaded. The untagged images were being cleaned up by harbor's garbage collection.
